src/risedevtool/src/task/compactor_service.rs

```diff
@@ -45,7 +45,19 @@ impl CompactorService {
     }
 
     /// Apply command args according to config
-    pub fn apply_command_args(cmd: &mut Command, config: &CompactorConfig) -> Result<()> {
+    pub fn apply_command_args(
+        cmd: &mut Command,
+        config: &CompactorConfig,
+        hummock_in_memory_strategy: HummockInMemoryStrategy,
```

Comment:
Currently, this is always `Disallowed`?

Reply:
This is always `Shared`. 

https://github.com/risingwavelabs/risingwave/blob/5da5ad6dc5119229d0e2968b8f258b81bf422bad/src/risedevtool/src/task/compactor_service.rs#L75

Reply:
I'm not sure why we are using hard-coded arguments for playground... But as explained in https://github.com/risingwavelabs/risingwave/pull/7423#discussion_r1071262796, theoretically it should be `Shared` for playground, so I kept it.